addon/components/nrg-modal-container.js

```diff
@@ -1,63 +1,12 @@
-import Component from '@ember/component';
-import { computed } from '@ember/object';
-import { readOnly } from '@ember/object/computed';
+import Component from '@glimmer/component';
 import { inject as service } from '@ember/service';
-import layout from '../templates/components/nrg-modal-container';
+import { action } from '@ember/object';
+export default class Modal extends Component {
+  @service('modal') modalService;
 
-export default Component.extend({
-  layout,
-  tagName: '',
-
-  modalService: service('modal'),
-
-  openModals: readOnly('modalService.openModals'),
-
-  activeModal: readOnly('modalService.activeModal'),
-
-  hasOpenModals: readOnly('modalService.hasOpenModals'),
-
-  activeModalIsSidebar: readOnly('activeModal.sidebar'),
-
-  _containerClass: computed('hasOpenModals', 'activeModalIsSidebar', function() {
-    let appliedClass = '';
-    if (this.hasOpenModals) {
-      appliedClass += ' visible active';
-    } else {
-      appliedClass += ' hidden';
-    }
-    if (this.activeModalIsSidebar) {
-      appliedClass += ' sidebar-container';
-    }
-    return appliedClass;
-  }),
-
-  dimmerClass: computed(
-    'activeModal.sidebar',
-    'activeModal.dimmerClass',
-    'activeModal.dismissable',
-    function() {
-      const activeModal = this.activeModal;
-      if (!activeModal) {
-        return '';
-      }
-      let appliedClass = '';
-      if (activeModal.sidebar) {
-        appliedClass += ' sidebar-dimmer';
-      } else {
-        appliedClass += ' page';
-      }
-      if (!activeModal.dismissable) {
-        appliedClass += ' not-dismissable';
-      }
-      appliedClass += ' ' + activeModal.dimmerClass;
-      return appliedClass;
-    }
-  ),
-
-  onDimmerClick() {
-    if (!this.activeModal.dismissable) {
-      return;
+  @action onDimmerClick() {
+    if (this.modalService?.activeModal?.dismissable) {
+      this.modalService?.activeModal?.onHide();
     }
-    this.activeModal.onHide();
-  },
-});
+  }
+};
```

Comment:
My eslint does not like this extra semicolon.
![image](https://user-images.githubusercontent.com/2756533/140997753-59ec4771-af65-4af7-8a8a-b52553fc2cdf.png)
Does yours call this out?
```suggestion
}
```

Reply:
Mine does not. I'll look into why.